src/backend/metal/CommandBufferMTL.mm

```diff
@@ -129,11 +113,9 @@ void BeginSubpass(id<MTLCommandBuffer> commandBuffer, uint32_t subpass) {
                         descriptor.stencilAttachment.texture = texture;
                         descriptor.stencilAttachment.storeAction = MTLStoreActionStore;
 
-                        bool shouldClearStencilOnFirstUse =
```

Comment:
I'm guessing this optimization is no longer applicable because subpasses are gone, so you always need to apply the loadop?

Reply:
This was because the semantics of multi pass render passes were that attachments that need to be cleared are cleared at the beginning of the render pass so we cleared that on the first subpass they were used. This isn't needed anymore because there's only ever a single subpass that's the first one.